src/Popover.jsx

```diff
@@ -139,6 +139,8 @@ class Popover extends React.Component {
 		const {
 				trigger,
 				menuItems, // eslint-disable-line no-unused-vars
+				alignLeft,
+				alignRight,
```

Comment:
instead of two different alignment props, could we instead do something like this?

```
<Popover ... align='left' />
```

```
React.propTypes = {
     align: React.PropTypes.oneOf(['right', 'left'])
}
```

This would allow us to add more alignment types in the future (eg. 'center', 'bottom') and is more likely to match up with an "align" prop signature in other components.